src/components/landing-page/landing-page.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import './landing-page.scss';
+
+class LandingPage extends Component {
+
+  handleClick(e) {
```

Comment:
should this also be an arrow function for consistency?

Reply:
done